### ISSUE TITLE
accessibility.sty not found

### ISSUE THREAD

**Issue by royka000** [AndyClifton/accessibility#31]
TexnicCenter would not install the package on the fly, and when I installed it manually, it still gives the same error: accessibility.sty not found.  No other package is having this issue that I have tried.  I get the error from any file, such as the following one:

\documentclass{article}
\usepackage{accessibility}
\begin{document}

Something something

\end{document}

From ctan, <https://ctan.org/pkg/accessibility?lang=en>, it says the package should be available in miktex.  However, when the viewing the package in the repository, there is only accessibility.tmp, but no accessibility.sty file.  Is this the problem?

**Comment by AndyClifton** [comment]
@royka000 - there were some problems with the original version of accessibility in the CTAN repository. Can you confirm this bug is still current, please?

If it is, can you tell me more about your installation, please?

**Comment by viktoriasee** [comment]
I confirm the package is empty except from the doc. This should be trivial to crosscheck? I first thought that it's a Miktex packaging problem but of course if there are no files on ctan Miktex cannot package them.

**Comment by viktoriasee** [comment]
The file is there now, tested with MikTeX
